src/docs/component/ToggleButtonDocs.vue

```diff
@@ -19,6 +19,10 @@
           <p>
             <va-toggle v-model="toggle3" :busy="busy"></va-toggle>
           </p>
+          <h5>Custom color</h5>
+          <p>
+            <va-toggle v-model="toggle4" color="#ffa500"></va-toggle>
```

Comment:
I think it would be nice to change `color` to `va-color` to distinguish parameters passed directly to AUI component from the ones added by `vue-aui`